$NetBSD: patch-compiler-Makefile,v 1.2 2017/09/10 02:36:08 dholland Exp $

Install properly. As of 8.06.3 $(INSTALLDIR) seems to already contain
$(DESTDIR) in it, so don't insert it again.

--- compiler/Makefile.orig	2017-07-19 03:52:12.000000000 +0000
+++ compiler/Makefile
@@ -61,8 +61,8 @@ scratch :
 	rm -f *.cm*  pplex.ml ppyac.ml ppyac.mli pp$(EXE)
 
 install:
-	cp tkcompiler$(EXE) $(INSTALLDIR)
-	cp pp$(EXE) $(INSTALLDIR)
+	$(BSD_INSTALL_SCRIPT) tkcompiler$(EXE) $(INSTALLDIR)
+	$(BSD_INSTALL_SCRIPT) pp$(EXE) $(INSTALLDIR)
 
 .SUFFIXES :
 .SUFFIXES : .mli .ml .cmi .cmo .mlp
